libs/components/ag-grid/src/lib/styles/ag-grid-theme.ts

```diff
@@ -0,0 +1,31 @@
+import { SkyThemeSettings } from '@skyux/theme';
+
+type SkyAgGridThemeType =
+  `ag-theme-sky-${'data-entry-grid' | 'data-grid'}-${'default' | 'modern-light' | 'modern-dark' | 'modern-light-compact' | 'modern-dark-compact'}`;
+
+export function agGridThemeIsCompact(
```

Comment:
any reason you didn't want to add these functions to the ag grid service? Was it just getting bloated? I'm not opposed to this, just don't see us export helper functions individually too often

Reply:
I went with this file because I wanted to minimize the number of places with this logic without adding these functions to the public API. On the service, the methods would need to be public.